js/lib/d3-timeline.js

```diff
@@ -156,7 +156,7 @@
 
       if (showTimeAxis) {
         g.append("g")
-          .attr("class", "axis")
+          .attr("class", "axis x-axis")
```

Comment:
If we can prevent adding an "x-axis" class it would be better. Maybe you can get the first axis in the `divId` or something along those lines e.g. `$("#timeline > svg > .axis")[0]`